contrib/Detached.Mappers.EntityFramework.Contrib.SysTec/ComplexDbContext.cs

```diff
@@ -102,18 +110,30 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         public override int SaveChanges()
         {
             //var now = DateTime.Now;
+            List<ConcurrencyStampBase> entities = new List<ConcurrencyStampBase>();
             foreach (var changedEntity in ChangeTracker.Entries())
             {
                 if (changedEntity.Entity is ConcurrencyStampBase entity)
                 {
-                    // Simulate DB Trigger.
-                    // With postgresql we are using xmin column which gets updated automatically
-                    entity.ConcurrencyToken += 1;
```

Comment:
this simulation introduces errors as it changes the states being set up by the mapper and EF
there is a weird thing over there between OriginalValue and CurrentValue of concurrency token props
changed to an improved version below

Reply:
also unchanged entities shoulnt be updated
needed to fix 2, 3 and 12

Reply:
Please include intention comments to other developers, why you favoured this implementation. I think other developers don't take this comments from this pull request.